sdk/eventhub/azure-eventhub/tests/livetest/asynctests/test_receive_async.py

```diff
@@ -135,7 +135,8 @@ async def on_event(partition_context, event):
         ed = EventData("Event Number {}".format(i))
         ed.properties = app_prop
         event_list.append(ed)
-    senders[0].send(event_list)
+
+    senders[0].send(EventDataBatch._from_batch(event_list))
```

Comment:
I find it funny that in our tests (both EH and SB), we use these private shortcuts to create a batch...
I wonder how many customers do the same :P

Reply:
yea.. I remember there was once a discussion on enabling the Batch object to accept a list, however the pitfall here is when trying to adding a list into batch which exceeds the batch size, then users still have to loop over the list and add event one by one to avoid exceeding the size limit.